### PR TITLE
prepared files for assignment-1

### PR DESCRIPTION
**Pull request by manisha15726**
Implemented everything for assignment 1

### CODE REVIEW

**Comment by manisha15726** [comment]
Everything is fine, just merge

**Comment by manisha15726** [comment]
Everything is fine, just merge

File: Main.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+ 
+        
+        if (args.length > 0) {
+ 
+            
+            System.out.println(" "
+                               + " Argument");
+ 
+            
+            for (String val : args)
```
Comment (raaaimund) [inline Main.java]: You only should print the FIRST argument, ignore any further arguments.

File: Main.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+ 
+    
+    public static void main(String[] args)
+    {
+ 
+        
+        if (args.length > 0) {
+ 
+            
+            System.out.println(" "
```
Comment (raaaimund) [inline Main.java]: You should print the FIRST argument.

File: Main.java [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+                System.out.println(val);
+        }
+ 
+        else
+ 
+            
+            System.out.println(" "
+                               + "No argument given.");
+    }
+}
```
Comment (raaaimund) [inline Main.java]: There should be an empty new line at the end of each .java file. See https://stackoverflow.com/questions/4700312/empty-new-line-at-the-end-of-the-java-source-files for more information.

File: Main.java [hunk 4]
```diff
@@ -0,0 +1,27 @@
+public class Main{
```
Comment (raaaimund) [inline Main.java]: Only submit formatted source code!
VSCode: https://stackoverflow.com/questions/29973357/how-do-you-format-code-in-visual-studio-code-vscode
IntelliJ: https://www.jetbrains.com/help/idea/reformat-and-rearrange-code.html

File: Main.java [hunk 5]
```diff
@@ -2,12 +2,12 @@ class Main {
 	public static void main(String[] args)
 	{
 		if (args.length > 0) {
-			System.out.println(" " + " Argument");
+			System.out.println("Argument");
 			for (String val : args)
```
Comment (raaaimund) [inline Main.java]: Nice job until now, the only thing that is still missing is, that you should not print ALL arguments, you should only print the FIRST argument.

File: Main.java [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+             System.out.println(args[0]);
+           }
+        else
+             System.out.println("No arguments given");
+      }
+  }       
+	
+
+
+
```
Comment (raaaimund) [inline Main.java:16]: Remove the unnecessary empty new lines (except of 1 new line) at the end of the file.

File: Main.java [hunk 7]
```diff
@@ -0,0 +1,16 @@
+public class Main
+ {
+    public static void main(String[] args)
+      {
```
Comment (raaaimund) [inline Main.java:4]: Format your code again -> use the shortcut for VSCode or IntelliJ.

**Comment by raaaimund** [comment]
Nicely done @manisha15726, only 2 things are missing
- [x] format your code before commiting
- [x] remove unnecessary new lines (except for 1) at the end of the file.

**Comment by raaaimund** [comment]
If you do not want to implement the remaining open comments, please merge this branch to be ready for the next assignment. If you fix the remaining open comments I can also give you the extra points.

**Pull request merged**